models/articles-model.js

```diff
@@ -21,3 +21,22 @@ exports.fetchArticleById = (article_id) => {
         
     })
 }
+
+exports.fetchArticleCommentById = (article_id) => {
+    return db
+    .query(`SELECT * FROM articles WHERE article_id = $1;`, [article_id])
```

Comment:
You already have a function which does this! It's called fetchArticleById (probably). You can invoke this function which will handle a 404 for you. But the way you have it is fine!